src/plugins/pillbox-menus/components/Menu.tsx

```diff
@@ -195,6 +192,48 @@ export default class Menu extends Component<{
   }
 }
 
+function numberOption(
+  controller: PillboxMenus,
+  item: ConfigItem,
+  plugin: SpecificPlugin,
+  settings: GenericSettings
+) {
+  const numItem = item as ConfigItemNumber;
+
+  const inputHandler = (e: InputEvent) => {
+    const value = Number((e.target as HTMLInputElement)?.value);
+    if (!isNaN(value)) {
+      controller.expandedPlugin &&
+        controller.controller.setPluginSetting(
```

Comment:
Holding down the "up" arrow while inside the number input field causes "Uncaught (in promise) Error: This request exceeds the MAX_WRITE_OPERATIONS_PER_MINUTE quota."

The color picker (for set-primary-color) deals with this by treating oninput differently than onchange. But that's not enough here. Playing around with `data:text/html,<input type="number" oninput="console.log('oninput')" onchange="console.log('onchange')"/>`, I notice that clicking+holding the up/down arrows trigger `oninput` until the click is released. But the arrow keys continuously trigger both `oninput` and `onchange`.

So you need to manually do some sort of debouncing. Either here (in the number handler), or in the `postMessageUp` in `setPluginSetting`.

Reply:
Fixed this by rate-limiting `setPluginSetting`